source/docs/api-reference/api-usage/signal-logging.rst

```diff
@@ -211,3 +217,22 @@ An example error that may occur if the free space limit is reached is shown belo
 .. code-block:: text
 
    [phoenix] Signal Logger: Available disk space (3 MB) below 5 MB, stopping log
+
+Converting Signal Logs
+----------------------
+
+Signal logs can be converted to other common file formats such as WPILOG or MCAP using the :doc:`Tuner X Log Extractor </docs/tuner/tools/log-extractor>`.
```

Comment:
Is it WPILog or WPILOG? I've always used the former.

Reply:
WPILOG or wpilog. We use the all caps version everywhere else.